CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ FEATURES:
 
 ENHANCEMENTS:
 
-
+* Updated `rancher2_cluster_v2.rke_config.machine_selector_config.config` argument as optional/computed
```

Comment:
This can be dropped for two reasons: 1) this file is outdated, 3.0.1 was already released 2) we will add this info when releasing the next version.